src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java

```diff
@@ -17,15 +17,63 @@ public class MdBenefitsFlowJourneyTest extends AbstractBasePageTest {
     protected static final String RANGE_ERROR_MESSAGE = "Make sure to provide a value between 1 and 100.";
 
     @Test
-    void chooseProgramsFlow() {
-        testPage.navigateToFlowScreen("mdBenefitsFlow/choosePrograms");
+    void redirectMDThink() {
```

Comment:
Instead of the comment, wouldn't it be better to have 2 test methods - `redirectToMDThinkOnUnsupportedProgram` and `redirectToMDThinkOnUnsupportedCounty`?